src/handlers/pullRequest.ts

```diff
@@ -36,4 +36,28 @@ export default (app: Probot) => {
       await pr.clearLabels();
     }
   });
+
+  app.on(['pull_request.converted_to_draft'], async context => {
+    const pr = new PullRequest(context);
+
+    let title = pr.data.title;
+    if (pr.wip) {
+      title = title.substring(6);
+      await pr.setTitle(title);
+    }
+
+    await pr.addLabel('wip');
+  });
+
+  app.on(['pull_request.ready_for_review'], async context => {
+    const pr = new PullRequest(context);
+
+    let title = pr.data.title;
+    if (pr.wip) {
+      title = title.substring(6);
```

Comment:
Can we use a smarter regex replacement? It's likely what someone (those who use WIP in the title themsevles when they create PRs) will expand our wip regex  to match a few different wip prefixes. Like `WIP: ` but more specifically, to not require the whitespace. Or to have too much whitespace between the wip and title that they would expect trimmed off when we process it like this.